plugins/modules/devices.py

```diff
@@ -263,10 +264,28 @@ def chdev(module, device):
 
     if attributes:
         opts += "-a '"
+        skip_attr = False
         for attr, val in attributes.items():
-            opts += "%s=%s " % (attr, val)
+
+            if device == "inet0" and attr in\
```

Comment:
i am not familiar, but is this only specific to "inet0" or a specific type (or inet0 is the type)?

Reply:
Yes, only specific to inet0 (there are no other inetX possible network devices)